op-monitorism/transaction_monitor/monitor.go

```
@@ -0,0 +1,377 @@
+	block, err := m.client.BlockByNumber(ctx, big.NewInt(int64(blockNum)))
+	if err != nil {
+		return fmt.Errorf("failed to get block: %w", err)
+	}
+
+	m.blocksProcessed.WithLabelValues(fmt.Sprint(blockNum)).Inc()
+
+	for _, tx := range block.Transactions() {
+		from, err := types.Sender(types.NewLondonSigner(tx.ChainId()), tx)
+		if err != nil {
+			return fmt.Errorf("failed to find tx sender: %w", err)
+			m.unexpectedRpcErrors.WithLabelValues("monitor", "no tx sender").Inc()
```

golangci-lint (1.62.2)

unreachable: unreachable code

(govet)